### PR TITLE
Fixes #15770 - clean up actions moved to core

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by adamruzicka** [comment]
What's the status on this?

**Comment by ares** [comment]
waiting on https://github.com/theforeman/foreman/pull/3670 to be merged

**Comment by ohadlevy** [comment]
@ares @iNecas it has been merged already.

File: lib/foreman_tasks/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ForemanTasks
-  VERSION = "0.7.19"
+  VERSION = "0.8.0"
```
Comment (iNecas) [inline lib/foreman_tasks/version.rb]: Let's leave the version bumping outside of the PR

**Comment by ares** [comment]
Rebased and I removed version bump, @iNecas please take another look.

**Comment by adamruzicka** [comment]
@ares Needs a rebase

**Comment by ares** [comment]
I'll hold on with the rebase until we get the update, probably next week.

**Comment by adamruzicka** [comment]
@ares what's the status here?

**Comment by ares** [comment]
I think these should remain after the foreman-tasks was reverted. Closing.

**Pull request closed without merging**